src/plugins/workspace/public/utils.ts

```diff
@@ -55,3 +60,42 @@ export const featureMatchesConfig = (featureConfigs: string[]) => ({
 
   return matched;
 };
+
+/**
+ * Check if an app is accessible in a workspace based on the workspace configured features
+ */
+export function isAppAccessibleInWorkspace(app: App, workspace: WorkspaceObject) {
+  /**
+   * When workspace has no features configured, all apps are considered to be accessible
+   */
+  if (!workspace.features) {
+    return true;
+  }
+
+  /**
+   * The app is configured into a workspace, it is accessible after entering the workspace
+   */
+  const featureMatch = featureMatchesConfig(workspace.features);
```

Comment:
`featureMatch` or `featureMatcher`, I feel that "matcher" more accurately refers to the mechanism responsible for performing the evaluation. Just my two cents. :)

Reply:
@Flyingliuhub Updated :)